contributing/communications.md

```diff
@@ -38,10 +38,10 @@ We don't have _strict_ guidelines on how you should write. We're not going to at
 
 Follow Plain English guidelines to help safeguard the accessibility of your written content. It'd be ironic for an a11y project to produce documents that are hard to understand! All relevant guidelines for [writing well][writing-well] apply. You might find tools like [Hemingway App][hemingway] helpful when editing. In particular:
 
-- Be concise  
+- Be brief  
```

Comment:
One thing I would be wary of is substituting plain English for simple English, those two concepts aren't equivalent either. Here's a short explanation of the difference: http://www.clearest.co.uk/editorsoftware/plain-english/ 

I don't think "concise" is a complicated word for foreign language speakers, FWIW - it's directly translated as some variation of "concis" in most Western European languages and also a few Eastern European ones.